homeassistant/components/input_number/__init__.py

```diff
@@ -66,7 +66,7 @@ def _cv_input_number(cfg):
     vol.Required(CONF_MIN): vol.Coerce(float),
     vol.Required(CONF_MAX): vol.Coerce(float),
     vol.Optional(CONF_INITIAL): vol.Coerce(float),
-    vol.Optional(CONF_STEP, default=1): vol.All(vol.Coerce(float), vol.Range(min=1e-3)),
+    vol.Optional(CONF_STEP, default=1): vol.All(vol.Coerce(float), vol.Range(min=1e-9)),
```

Comment:
Does this work in the frontend ?

Reply:
Confirmed by @bramkragten it won't break fronted